src/DynamoApplications/PathResolvers.cs

```diff
@@ -96,11 +96,15 @@ public CLIPathResolver(string preloaderLocation)
                 "ProtoGeometry.dll",
                 "DesignScriptBuiltin.dll",
                 "DSCoreNodes.dll",
+                "DSOffice.dll",
```

Comment:
If we are sure about the performance impact, I see no harm adding these dlls and looks like the comment at line:92 needs to be removed 😉

Reply:
this was done initially for optimization workflows that benefited from the slightly faster startup time of ignoring these libraries. It's worth profiling it I think before making this change unless they are simply required for your work @sm6srw.